### PR TITLE
[TT-10732] Fixed edp probes

### PR DESCRIPTION
**Pull request by komalsukhani**
## Description

Fixed probes path of enterprise portal. Update enterprise portal version to latest.

## Related Issue

[TT-10732](https://tyktech.atlassian.net/browse/TT-10732)

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). If PRing from your fork, don't come from your `master`!
- [x] Make sure you are making a pull request against our **`master` branch** (left side). Also, it would be best if you started *your change* off *our latest `master`*.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

[TT-10732]: https://tyktech.atlassian.net/browse/TT-10732?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tyk-pro/templates/statefulset-enterprise-portal.yaml [hunk 1]
```diff
@@ -152,7 +152,7 @@ spec:
         livenessProbe:
           httpGet:
             scheme: "HTTP{{ if .Values.enterprisePortal.tls }}S{{ end }}"
-            path: /
+            path: /live
```
Comment (buraksekili) [inline tyk-pro/templates/statefulset-enterprise-portal.yaml:155]: Is this change backward compatible? What happens if users use 1.x.y which had no support for `/live` path? Should we make path field configurable?

Comment (komalsukhani) [reply tyk-pro/templates/statefulset-enterprise-portal.yaml:155]: That make sense. Will update PR accordingly.

Comment (komalsukhani) [reply tyk-pro/templates/statefulset-enterprise-portal.yaml:155]: @buraksekili Probes are added in portal v1.2.0 https://tyk.io/docs/product-stack/tyk-enterprise-developer-portal/release-notes/portal-1.2.0/

Comment (buraksekili) [reply tyk-pro/templates/statefulset-enterprise-portal.yaml:155]: thank you for the reference 🙏

**Review by buraksekili — COMMENTED** [review]
lgtm. can you please resolve the conflict in CHANGELOG?

**Review by buraksekili — APPROVED** [review]

**Pull request merged**